Sources/StreamChat/Models/Attachments/ChatMessageAttachment_Sample.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable
+import StreamChat
+
+extension ChatMessageAttachment {
+    static func sample(
+        id: AttachmentId = .unique,
+        type: AttachmentType = .image,
+        payload: Any? = nil,
+        uploadingState: AttachmentUploadingState? = nil
+    ) -> ChatMessageAttachment {
+        ChatMessageAttachment(
+            id: id,
+            type: type,
+            payload: payload,
+            uploadingState: uploadingState
+        )
+    }
+}
```

Comment:
Is this dummy data? We put these under `StreamChatTestTools > DummyData` and name them `dummy`

Reply:
I wanted to raise this with the team
if you think about it, it's not really a dummy, because it's a totally normal fully functioning object. So I wanted to suggest this little improvement to naming for a method that helps you to build this logic

Reply:
I would say the right technical naming for this kind of test doubles is a `stub`. A dummy is normally when you want to just pass data to conform to an interface but the values of the dummy don't really influence the test. For stubs, you pre-populate the data in order for a specific test to behave in some manner. Because this is so subjective and makes it hard to think about how to name your mocks/stubs etc... I just default everything to `.mock()`. In this case, because we are already using `dummy` everywhere, let's keep it like that for now, since it is the pattern that is currently in the project. We can create a tech debt ticket to change this tho after discussing it with the team 👍